tests/README.md

```diff
@@ -37,3 +37,45 @@ MOONBEAM_LOG="warn,rpc=trace" npm run test
 
 The test script will find available ports above 20000 in order to ensure that it doesn't conflict
 with any other running services.
+
+# Debugging a Moonbeam node
+
+The repository contains a pre-configured debugger configuration for VSCode with the **CodeLLDB**
+(`vadimcn.vscode-lldb`) extension.
+
+Before debugging, you need to build the node with debug symbols enabled by running
+`./scripts/debugger-build.sh`. Then go in the **Debug** tab in the left bar of VSCode and make sure
+**Launch Moonbeam Node (Linux)** is selected in the top dropdown.
+
+To launch the debug session click on the green "play" arrow next to the dropdown. It will take some
+time before the node starts, but the terminal containing the node output will appear when it is
+really starting.
+
+You can explore the code and place a breakpoint on a line by left clicking on the left of the line
+number. The execution will pause the next time this line is reached. The debug toolbar contains the
+following buttons :
+
+- Resume/Pause : Resume the execution if paused, pause the execution at the current location
+  (pretty random) if running.
+- Step over : Resume the execution until next line, or go one level up if the end of the current
+  scope is reached.
+- Step into : Resume the execution to go inside the immediatly next function call if any, otherwise
+  step to next line.
+- Step out : Resume the execution until the end of the scope is reached.
+- Restart : Kill the program and start a new debuging session.
+- Stop : Kill the program and end debugin session.
+
+Breakpoints stay between debugging sessions. When multiple function calls are made on the same line,
+multiple step into, step out, step into, ... can be requiered to go inside one of the chained
+calls.
+
+When paused, content of variables is showed in the debuging tab of VSCode. Some basic types are
+displayed correctly (primitive types, Vec, Arc) but more complex types such as HashMap/BTreeMap
+are not "smartly" displayed (content of the struct is shown by mapping is hidden in the complexity
+of the implementation).
+
+## Running Typescript tests with a debug node
+
+By setting the environement variable `DEBUG_MODE=true`, the Typescript tests will not spawn its
```

Comment:
if this flag is necessary to debug, it should be included in a command in a package.json

Reply:
How do set an environment variable in package.json ? Otherwise I can add a task in VSCOde tasks.json.

Reply:
`"test-debug":"DEBUG_MODE=true npm run test"

Reply:
Added a VSCode task for that and more explanations. Only one test file should be ran, then the node should be restarted manually.